libs/domain/checkout/src/mocks/src/mock-checkout.providers.ts

```diff
@@ -1,33 +1,50 @@
 import {
-  CheckoutDataService,
-  CheckoutOrchestrationService,
   CheckoutPaymentService,
+  CheckoutProcessState,
   CheckoutService,
   CheckoutShipmentService,
-  DefaultCheckoutDataService,
-  DefaultCheckoutOrchestrationService,
 } from '@spryker-oryx/checkout';
 import { Provider } from '@spryker-oryx/di';
-import { MockCheckoutService } from './mock-checkout.service';
+import { RouterService } from '@spryker-oryx/router';
+import { SemanticLinkService } from '@spryker-oryx/site';
+import { of } from 'rxjs';
+import { vi } from 'vitest';
```

Comment:
not needed

Reply:
we have TS issues if we drop it

Reply:
Wow, then wait a sec. Why do we need to use `vitest` for mocking data? Just:

```
export class MockCheckoutService implements Partial<CheckoutService> {
  register = () => undefined;
  getProcessState = () => of(CheckoutProcessState.Initializing);
  placeOrder = () => of();
}
```

and etc...

Reply:
yeah my bad, will clean it up as it doesn't belong here